landscape_setup/monitoring.py

```diff
@@ -147,3 +163,52 @@ def create_postgresql_helm_values(
         }
     }
     return helm_values
+
+
+def deploy_resurrector(cfg_set_name, monitoring_namespace):
+    name = 'worker-resurrector'
+    labels= {'app': name}
+
+    resurrector_deployment = V1Deployment(
+        kind='Deployment',
+        metadata=V1ObjectMeta(
+            name=name,
+            labels=labels,
+        ),
+        spec=V1DeploymentSpec(
+            replicas=1,
+            selector=V1LabelSelector(match_labels=labels),
+            template=V1PodTemplateSpec(
+                metadata=V1ObjectMeta(labels=labels),
+                spec=V1PodSpec(
+                    containers=[
+                        V1Container(
+                            image='eu.gcr.io/gardener-project/cc/job-image:latest',
+                            image_pull_policy='Always',
+                            name=name,
+                            command=['python3'],
```

Comment:
cannot /cc/utils/cli.py be the executable? this would have the slight benefit of not hard-coding it's a python3 application

Reply:
done